src/notion/util.js

```diff
@@ -104,7 +104,7 @@ const formatLWD = (tasks) => {
 
           const link = taskDetails.href ? { url: taskDetails.href } : null
           if (i === 0) {
-            const newLine = index ? '\n' : ''
+            const newLine = index || s ? '\n' : ''
```

Comment:
though the language has built-in operator precedence, I think for the sake of reading/reasoning the code, it's a good practice to highlight the sequence of operation with parentheses `(index || s) ? '\n' : ''` (unless it's actually intended as `index || (s ? '\n' : '')`, in which case do either way 😂)

Reply:
ah its supposed to be `(index || s) ? '\n' : ''` 🌚